src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,3 @@
+* {
+  margin: 0;
+}
```

Comment:
Will be better not to set margin: 0 to all elements
Just specify needed elements, for which you need to reset margins